config/routes.rb

```diff
@@ -16,7 +16,9 @@
   post '/api/usps_locations' => 'idv/in_person/public/usps_locations#index'
   match '/api/usps_locations' => 'idv/in_person/public/usps_locations#options', via: :options
 
-  post '/api/webhooks/socure/event' => 'socure_webhook#create'
+  if IdentityConfig.store.socure_webhook_enabled
+    post '/api/webhooks/socure/event' => 'socure_webhook#create'
+  end
```

Comment:
I think we'd be better off using something like this in the controller so that we don't have to worry about reloading routes and stuff, the code will return a 404 so it's the same result to users

```
check_or_render_not_found -> { IdentityConfig.store.in_person_proofing_enabled }
```

Reply:
Based on @zachmargolis's comment, I'd imagine we'd remove the condition here now that we're checking in the controller itself.

```suggestion
  post '/api/webhooks/socure/event' => 'socure_webhook#create'
```